core/modules/layout/tests/layout.test

```diff
@@ -1981,9 +1981,7 @@ class LayoutBlockTest extends BackdropWebTestCase {
     $hero_title_1 = 'Wonderwoman';
     $hero_content_1 = 'Youth is your greatest weapon, your greatest tool.';
     $hero_title_2 = 'Black Widow';
-    // "isnt" is purposefully missing an apostrophe as it was causing tests to
-    // fail.
-    $hero_content_2 = "This isnt gonna end well...";
+    $hero_content_2 = "Whatever it takes.";
```

Comment:
I had a look in https://everydaypower.com/black-widow-quotes, and I'd like to suggest this instead:
```suggestion
    $hero_content_2 = "At some point, we all have to choose between what the world wants you to be and who you are.";
```

Reply:
...or this shorted one:
```suggestion
    $hero_content_2 = "It was real to me too.";
```

Reply:
I don't think we really need to bikeshed on the wording for block content *in a test*. Request denied 😝